docs/actions/gcp-gsm-load-secrets/README.md

```diff
@@ -0,0 +1,57 @@
+# gcp-gsm-load-secrets
+
+This action is set to replace GitHub actions integrated secret management.
+
+## Usage
+
+To load a secret from GSM figure out the following:
+
+- check if the repository has access to the secret
+  - repository is owned by bakdata
+  - repository is private
+  - even if the labels are correctly set, you will need to run Terraform to set the proper roles
+  - _TBD_
```

Comment:
Do you plan to keep the TBD or did you forget to add something here?

Reply:
TBD was meant for any additional constraints we might want to set